### PR TITLE
doc: create add/remove metadata server

### PR DESCRIPTION
**Pull request by joscollin**
A new doc is created based on https://github.com/ceph/ceph/pull/22035#pullrequestreview-121600153.

Fixes: http://tracker.ceph.com/issues/24093
Signed-off-by: Jos Collin <jcollin@redhat.com>

### CODE REVIEW

**Comment by joscollin** [comment]
Jenkins render docs

**Comment by joscollin** [comment]
Jenkins render docs

**Comment by joscollin** [comment]
Jenkins render docs

**Comment by joscollin** [comment]
Jenkins render docs

File: doc/cephfs/add-remove-mds.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Add a Metadata Server
+=====================
+
+#. Create an mds data point ``/var/lib/ceph/mds/mds.<your-mds-id>``.
+
+#. Edit ``ceph.conf`` and add MDS sections like: ::
+
+	[mds]
+	mds data = /var/lib/ceph/mds/mds.<your-mds-id>
+	keyring = /var/lib/ceph/mds/mds.<your-mds-id>/mds.<your-mds-id>.keyring
```
Comment (ukernel) [inline doc/cephfs/add-remove-mds.rst]: these are default values, no need to add these configs

File: doc/cephfs/add-remove-mds.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+	[mds.0]
+	host = {hostname}
+
+#. Create the authentication key, if you use CephX. ::
+
+	$ sudo ceph auth get-or-create mds.<your-mds-id> mds 'allow ' osd 'allow *' mon 'allow rwx' > /var/lib/ceph/mds/mds.<your-mds-id>/mds.<your-mds-id>.keyring
+
```
Comment (ukernel) [inline doc/cephfs/add-remove-mds.rst]: ceph auth get-or-create mds.<your-mds-id> mon 'profile mds' mgr 'profile mds'  mds 'allow *' osd 'allow *' >/var/lib/ceph/mds/ceph-<your-mds-id>/keying

**Comment by joscollin** [comment]
Jenkins render docs

**Comment by joscollin** [comment]
@ukernel Please check now: http://docs.ceph.com/ceph-prs/22338/cephfs/add-remove-mds/

**Comment by ukernel** [comment]
LTGM

**Pull request merged**